de.dlr.sc.virsat.model.extension.funcelectrical.test/src/de/dlr/sc/virsat/model/extension/funcelectrical/excel/importer/FuncElecImporterTest.java

```diff
@@ -31,51 +31,48 @@
 
-		FuncElecImporter ei = new FuncElecImporter();
-		
-		List<Fault> faults = ei.validate(itc.getStructuralElementInstance(), wb);
+		List<Fault> faults = feImporter.validate(ifaceTypeCollection.getStructuralElementInstance(), wb);
 		assertTrue("Succesfully called the validator", faults.isEmpty());
-		
-		ei.importExcel(itc.getStructuralElementInstance(), repository, wb);
-		assertEquals(THREE, itc.getStructuralElementInstance().getCategoryAssignments().size());
-		assertEquals("FILL", itc.getStructuralElementInstance().getCategoryAssignments().get(0).getName());
-		assertEquals("HILL", itc.getStructuralElementInstance().getCategoryAssignments().get(1).getName());
-		assertEquals("PILL", itc.getStructuralElementInstance().getCategoryAssignments().get(2).getName());
+
+		feImporter.importExcel(ifaceTypeCollection.getStructuralElementInstance(), repository, wb);
+		assertEquals(NUMBEROFINTERFACES, ifaceTypeCollection.getStructuralElementInstance().getCategoryAssignments().size());
+		assertEquals("FILL", ifaceTypeCollection.getStructuralElementInstance().getCategoryAssignments().get(0).getName());
+		assertEquals("HILL", ifaceTypeCollection.getStructuralElementInstance().getCategoryAssignments().get(1).getName());
+		assertEquals("PILL", ifaceTypeCollection.getStructuralElementInstance().getCategoryAssignments().get(2).getName());
 	}
 
 	@Test
 	public void testImportInterfaceEnds() throws IOException {
-		InputStream is = Activator.getResourceContentAsString("/resources/ImportElementDefinitionTest.xlsx");
-		XSSFWorkbook wb = new XSSFWorkbook(is);
+		InputStream iStream = Activator.getResourceContentAsString("/resources/ImportElementDefinitionTest.xlsx");
+		XSSFWorkbook wb = new XSSFWorkbook(iStream);
+
+		repository.getRootEntities().add(ifaceTypeCollection.getStructuralElementInstance());
+		FuncElecImporter feImporter = new FuncElecImporter();
 
-		repository.getRootEntities().add(itc.getStructuralElementInstance());
-		FuncElecImporter ei = new FuncElecImporter();
-		
-		List<Fault> faults = ei.validate(ed.getStructuralElementInstance(), wb);
+		List<Fault> faults = feImporter.validate(elementDef.getStructuralElementInstance(), wb);
 		assertTrue("Succesfully called the validator", faults.isEmpty());
-		
-		ei.importExcel(ed.getStructuralElementInstance(), repository, wb);
+
+		feImporter.importExcel(elementDef.getStructuralElementInstance(), repository, wb);
 		BeanCategoryAssignmentHelper bCaHelper = new BeanCategoryAssignmentHelper();
-		List<InterfaceEnd> seiInterfaceEnds = bCaHelper.getAllBeanCategories(ed.getStructuralElementInstance(), InterfaceEnd.class);
-		assertEquals(THREE, ed.getStructuralElementInstance().getCategoryAssignments().size());
+		List<InterfaceEnd> seiInterfaceEnds = bCaHelper.getAllBeanCategories(elementDef.getStructuralElementInstance(), InterfaceEnd.class);
+		assertEquals(NUMBEROFINTERFACES, elementDef.getStructuralElementInstance().getCategoryAssignments().size());

```

Comment:
We use `_` for separating single words in constant names. So `NUMBEROFINTERFACES` should be `NUMBER_OF_INTERFACES`.